content/600-about/200-prisma-docs/20-style-guide/03-spelling-punctuation-formatting.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: 'Spelling, punctuation, and formatting'
+metaTitle: 'Prisma docs style guide: spelling, punctuation, and formatting'
+metaDescription: 'This section of the style guide provides guidelines on how to present the information you add.'
+tocDepth: 3
+---
+
+## Avoid contractions
+
+In keeping with our conversational style, some contractions are OK:
+
+- Contractions that include the word "not", such as can't, didn't, and doesn't.
```

Comment:
Hmmm I am wondering if this is the one contraction that we want to specifically avoid.... for non-native English speakers, it might be important to show the full word "not."

Reply:
I don't have a strong view on this, and these recommendations were either in the style guide already, or in the external guide that you recommended I look at. I don't remember this as a guideline when I was trained in docs for non-native speakers, but I might have forgotten it. I think if we're in any doubt, in this case we should advise against using this form.